### ISSUE TITLE
Custom forms don't support selector property type

### ISSUE THREAD

**Issue by rarsan** [cf-platform-eng/tile-generator#141]
[Tile generator docs](http://cf-platform-eng.github.io/isv-portal/tile-generator/) specify that all property types listed in [Ops Manager Product Template Reference](https://docs.pivotal.io/partners/product-template-reference.html#simple-vs-complex) are supported. However, `selector` property type does not seem to be supported.

**Comment by guidowb** [comment]
Form presentation has been fixed on a branch, but without special treatment of the `value`, the environment variable looks like this:

```
cf set-env app1-0.0.19 EXAMPLE_SELECTOR {"selected_option":"#<OpenStruct rarity_dropdown=\"medium\">","value":"Filet Mignon"}
```

Working on fixing that.

**Comment by guidowb** [comment]
Fixed by 27f58e6c599264adf4e24d7f53019e3dbc83f2e6
